ecommerce/cybersource_api.py

```diff
@@ -0,0 +1,99 @@
+"""Wrapper class for CyberSource"""
+
+import json
+import logging
+
+from django.db import transaction
+
+from ecommerce.models import Order, FulfilledOrder, Transaction
+from main.settings import ECOMMERCE_DEFAULT_PAYMENT_GATEWAY
+from ecommerce.constants import TRANSACTION_TYPE_REFUND, REFUND_SUCCESS_STATES
+from mitol.payment_gateway.api import (
+    Refund as GatewayRefund,
+    PaymentGateway,
+)
+from mitol.payment_gateway.exceptions import RefundDuplicateException
+
+log = logging.getLogger(__name__)
+
+
+def refund_order(*, order_id: int, **kwargs):
```

Comment:
I don't think this should be part of `cybersource_api.py` since it should ideally be generic.